service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -162,6 +162,13 @@ public void resetForResource(final String zoneId, final String resourceId) {
     }
 
     @Override
+    public void resetForResourcesByIds(final String zoneId, final Set<String> resourceIds) {
+        for (String resourceId : resourceIds) {
```

Comment:
@amiri-ge, good catch.  We pipelined cache reset for multiple resources and subjects using multiSet().   Refer to [multisetForResources](https://github.com/predix/acs/blob/6f34427429ce0f8ce829e5a63e78e1471e19b505/service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java#L176) and [multisetForSubjects](https://github.com/predix/acs/blob/6f34427429ce0f8ce829e5a63e78e1471e19b505/service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java#L224)